src/feature/control/control_events.c

```diff
@@ -1774,31 +1775,25 @@ control_event_guard(const char *nickname, const char *digest,
  * a smartlist_t containing (key, value, ...) pairs in sequence.
  * <b>value</b> can be NULL. */
 int
-control_event_conf_changed(const smartlist_t *elements)
+control_event_conf_changed(config_line_t *elements)
 {
-  int i;
   char *result;
   smartlist_t *lines;
-  if (!EVENT_IS_INTERESTING(EVENT_CONF_CHANGED) ||
-      smartlist_len(elements) == 0) {
+  if (!elements) {
```

Comment:
This seems to have deleted the `!EVENT_IS_INTERESTING(EVENT_CONF_CHANGED)` without replacing it.